cmd/mdatagen/internal/sampleconnector/internal/metadata/generated_config.go

```diff
@@ -9,11 +9,17 @@ import (
 
 // MetricConfig provides common config for a particular metric.
 type MetricConfig struct {
-	Enabled bool `mapstructure:"enabled"`
+	Enabled  bool   `mapstructure:"enabled"`
+	AggStrat string `mapstructure:"aggregation_strategy"`
```

Comment:
Let's keep it full name
```suggestion
	AggregationStrategy string `mapstructure:"aggregation_strategy"`
```

Reply:
We should also generate Validate method for MetricConfig to check for correct AggregationStrategy values